packages/connectivity_plus/connectivity_plus/ios/Classes/PathMonitorConnectivityProvider.swift

```diff
@@ -28,11 +28,11 @@ public class PathMonitorConnectivityProvider: NSObject, ConnectivityProvider {
 
   override init() {
     super.init()
-    ensurePathMonitor()
+    _ = ensurePathMonitor()
```

Comment:
Is this a good Swift practice? (I am not an experienced Swift dev, so I cannot tell)

Reply:
I never wrote Swift before. I looked for how to supress the warning as we can't remove that call from the plugin. Found 2 approaches: use underscore or use discardResult annotation.

Reply:
https://forums.swift.org/t/how-to-turn-off-unused-variable-warnings/47356/2

https://useyourloaf.com/blog/swift-3-warning-of-unused-result/

https://www.hackingwithswift.com/example-code/language/how-to-ignore-return-values-using-discardableresult

Reply:
thanks, makes sense! Swift is interesting sometimes :)

Reply:
Coool!